.github/workflows/integration-tests.yaml

```diff
@@ -295,3 +295,49 @@ jobs:
         with:
           name: tests-logs
           path: artifacts
+
+  integration-tests-legacy-v2:
+    name: Integration Tests Legacy V2
+    runs-on: ubuntu-20.04
+    needs: prepare-integration-tests
+    steps:
+      - name: Apicurio Registry (Version 2.5.x) Checkout
+        run: |
+          mkdir registry-v2
+          cd registry-v2
+          git init
+          git config --global user.name "apicurio-ci"
+          git config --global user.email "apicurio.ci@gmail.com"
+          git remote add origin "https://apicurio-ci:${{ secrets.ACCESS_TOKEN }}@github.com/Apicurio/apicurio-registry.git"
+          git fetch --all
+          git checkout 2.5.x
+          git branch --set-upstream-to=origin/2.5.x
+          git pull
+
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'maven'
+
+      - name: Run Legacy Integration Tests (Core API v2)
+        run: |
+          echo "Starting Registry App (In Memory)"
+          docker run -it -p 8181:8080 -d --env CORS_ALLOWED_ORIGINS=* ttl.sh/${{ github.sha }}/apicurio/apicurio-registry:1d
```

Comment:
The fact that we have to open up the CORS settings is a little suspicious, do we have a sensible default? Why it doesn't work in CI?

Reply:
Good point.  This isn't necessary because the CORS setting is only needed for the UI tests (another copy/paste issue).  But **also** I think you may be right that we don't need the setting at all anymore, even for the UI tests.  I'll explore that.

Reply:
Fixed here **and** in the UI tests section.  It might have been necessary originally, but at some point I changed the default value for `CORS_ALLOWED_ORIGINS` to `http://localhost:8888,http://127.0.0.1:8888` so it's no longer needed if you run the UI on port 8888 and test locally.